obvious-ci.docker/linux64_obvci/Dockerfile

```diff
@@ -35,18 +35,19 @@ RUN curl -s -L -O https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x8
     conda config --set show_channel_urls True && \
     conda update --all --yes && conda clean -tipsy
 
-# Add conda and friends to the path.
-ENV PATH /opt/conda/bin:$PATH
-
 # Install Obvious-CI.
-RUN conda install --yes -c pelson/channel/development obvious-ci && \
+RUN export PATH="/opt/conda/bin:${PATH}" && \
```

Comment:
You are still adding conda to the path here. I guess the thing you are trying to address is the addition of the activate step, right?